### PR TITLE
RavenDB-15680 Allow to Export/import the Hub/Sink Replication Task De…

### PR DESCRIPTION
**Pull request by Danielle9897**
…finition w/o Access Info - Studio side only

### CODE REVIEW

File: src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs [hunk 1]
```diff
@@ -54,6 +53,8 @@ public enum DatabaseRecordItemType
         SinkPullReplications = 1 << 13,
         HubPullReplications = 1 << 14,
         TimeSeries = 1 << 15,
-        DocumentsCompression = 1 << 16
+        DocumentsCompression = 1 << 16,
+        IncludeHubReplicationAccessInfo = 1 << 17,
```
Comment (ml054) [inline src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs]: Don't we need other changes in server side code to handle that? I can see this is only cs file touched in this PR.

Comment (ml054) [reply src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs]: ok answer is here: https://issues.hibernatingrhinos.com/issue/RavenDB-15680#focus=Comments-67-51338.0-0

File: src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs [hunk 2]
```diff
@@ -27,10 +27,9 @@ public enum DatabaseItemType
         CounterGroups = 1 << 15,
         Subscriptions = 1 << 16,
         CompareExchangeTombstones = 1 << 17,
-        
-
         TimeSeries = 1 << 18,
-        
+
+        [Obsolete("Please use DatabaseRecordItemType.IncludeHubReplicationAccessInfo instead.")]
         ReplicationHubCertificates = 1 << 19
```
Comment (ayende) [inline src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs]: Why is this Obsolete? This was never released, we can simple remove it, no?

File: src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs (hunk 1 above)
Comment (ppekrol) [reply src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs]: Why the name contains `Include...` ?

**Comment by Danielle9897** [comment]
Currently, this pr contains only changes for issue https://issues.hibernatingrhinos.com/issue/RavenDB-15665

**Review by ml054 — APPROVED** [review]

**Pull request merged**